File: app/src/main/java/com/acework/shabaretailer/model/Retailer.java
package com.acework.shabaretailer.model;

public class Retailer {
    private String name, businessName, telephone, county, street, email;
    private int lookbook;

    public Retailer() {
    }

    public Retailer(String name, String businessName, String telephone, String county, String street, String email) {
        this.name = name;
        this.businessName = businessName;
        this.telephone = telephone;
        this.county = county;
        this.street = street;
        this.email = email;
        this.lookbook = 0;
    }

    public String getName() {
        return name;
    }

    public String getBusinessName() {
        return businessName;
    }

    public String getTelephone() {
        return telephone;
    }

    public String getCounty() {
        return county;
    }

    public String getStreet() {
        return street;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public int getLookbook() {
        return lookbook;
    }
}

File: app/src/main/java/com/acework/shabaretailer/model/Retailer.kt
package com.acework.shabaretailer.model

data class Retailer(
    val businessName: String = "",
    val city: String = "",
    val country: String = "",
    val countryCode: String = "",
    val email: String = "",
    val id: String = "",
    val name: String = "",
    val number: String = "",
    val postalCode: String = "",
    val postalAddress: String = "",
    val stateCode: String="",
) {
    companion object {
        fun create(
            businessName: String,
            city: String,
            country: String,
            countryCode: String,
            email: String,
            id: String,
            name: String,
            number: String,
            postalCode: String,
            postalAddress: String,
            stateCode: String
        ): Retailer {
            return Retailer(
                businessName,
                city,
                country,
                countryCode,
                email,
                id,
                name,
                number,
                postalCode,
                postalAddress,
                stateCode
            )
        }
    }
}